packages/mcap-support/src/parseChannel.ts

```diff
@@ -74,12 +75,26 @@ export function parseChannel(channel: Channel): ParsedChannel {
         channel.schema.name,
       );
       datatypes = parsedDatatypes;
-      deserializer = (data) =>
+      deserializer = (data) => {
         postprocessValue(JSON.parse(textDecoder.decode(data)) as Record<string, unknown>);
+      };
     }
     return { fullSchemaName: channel.schema.name, deserializer, datatypes };
   }
 
+  if (channel.messageEncoding === "flatbuffer") {
+    if (channel.schema?.encoding !== "flatbuffer") {
```

Comment:
```suggestion
  if (channel.messageEncoding === "flatbuffers") {
    if (channel.schema?.encoding !== "flatbuffers") {
```

Seems like the official name is "flatbuffers" and I don't see this shortened version anywhere?

Reply:
Oops, my MCAP file generator was using the wrong name....

Reply:
We should probably change that, the flatbuffers docs seem to call them flatbuffers. cc @amacneil

Reply:
I don't feel strongly. I can find places in the docs where they say things like `the FlatBuffer compiler`. Protobuf is also called `Protocol buffers` in most places. As long as the mcap docs match our behavior.

Reply:
In [this list](https://github.com/foxglove/mcap/blob/main/docs/specification/appendix.md#well-known-message-encodings) of `well known message encodings`, they are all singular so it seems like making `flatbuffers` randomly plural would look a bit out of place. But I don't feel strongly about it.

Reply:
> In [this list](https://github.com/foxglove/mcap/blob/main/docs/specification/appendix.md#well-known-message-encodings) of well known message encodings, they are all singular so it seems like making flatbuffers randomly plural would look a bit out of place

didn’t we just make up this list recently though?

Reply:
they have all existed at least as long as the mcap spec being finalized. before that I can't remember.

Reply:
Ok, I see the docs do sometimes use "flatbuffer" so this is fine.

https://google.github.io/flatbuffers/flatbuffers_guide_tutorial.html

> - Write a FlatBuffer schema file.
> - Use the flatc FlatBuffer compiler.
> - Parse [JSON](http://json.org/) files that conform to a schema into FlatBuffer binary files.